tests/clpy_tests/core_tests/test_ndarray_complex_ops.py

```diff
@@ -59,12 +59,17 @@ def test_real_setter(self, xp, dtype):
         x.real = testing.shaped_reverse_arange((2, 3), xp, dtype).real
         return x
 
-    @testing.for_dtypes('FD')
-    @testing.numpy_clpy_array_almost_equal(accept_error=False)
-    def test_imag_setter(self, xp, dtype):
-        x = testing.shaped_arange((2, 3), xp, dtype)
-        x.imag = testing.shaped_reverse_arange((2, 3), xp, dtype).real
-        return x
+'''
+The reason for comment out is that ClPY does not correspond tocomplex
```

Comment:
* ClPy is not CLPY.
* correspond...? Do you mean "support"?
* tocomplex -> to complex

Reply:
I modified these problem.